community-membership.md

```diff
@@ -41,58 +41,24 @@ Members are expected to remain active contributors to the community.
     - Contributing to SIG, subproject, or community discussions (e.g. meetings, Slack, email discussion
       forums, Stack Overflow)
 - Subscribed to [kubernetes-dev@googlegroups.com]
+- Have read the [developer guide]
```

Comment:
I think I would rather see us link to the [Contributor's Guide](https://github.com/kubernetes/community/tree/master/contributors/guide) instead.  The devel directory is still a bit of a mess